receiver/elasticsearchreceiver/metadata.yaml

```
@@ -1 +1,301 @@
 name: elasticsearchreceiver
+
+attributes:
+  elasticsearch.cluster.name:
+    description: The name of the elasticsearch cluster.
+  elasticsearch.node.name:
+    description: The name of the elasticsearch node.
+  cache_name:
+    description: The name of cache.
+    enum:
+    - fielddata
+    - query
+  fs_direction:
+    value: direction
+    description: The direction of filesystem IO.
+    enum:
+    - read
+    - write
+  generation:
+    description: The generation on which garbage collection was performed.
+    enum:
+    - young
+    - old
+  disk_usage_state:
+    value: state
+    description: The state of a section of space on disk.
+    enum:
+    - used
+    - free
+  direction:
+    description: The direction of network data.
+    enum:
+    - received
+    - sent
+  document_state:
+    value: state
+    description: The state of the document.
+    enum:
+    - active
+    - deleted
+  shard_type:
+    value: type
+    description: The state of the shard.
+    enum:
+    - active
+    - relocating
+    - initializing
+    - unassigned
+  operation:
+    value: operation
+    description: The type of operation.
+    enum:
+    - index
+    - delete
+    - get
+    - query
+    - fetch
+    - scroll
+    - suggest
+    - merge
+    - refresh
+    - flush
+    - warmer
+  thread_pool_name:
+    description: The name of the thread pool.
+  thread_state:
+    value: state
+    description: The state of the thread.
+    enum:
+    - active
+    - idle
+  task_state:
+    value: state
+    description: The state of the task.
+    enum:
+    - rejected
+    - completed
+metrics:
+  # these metrics are from /_nodes/stats, and are node level metrics
+  elasticsearch.node.cache.memory.usage:
+    description: The size in bytes of the cache.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [cache_name]
+    enabled: true
+  elasticsearch.node.cache.evictions:
+    description: The number of evictions from the cache.
+    unit: "{evictions}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [cache_name]
+    enabled: true
+  elasticsearch.node.jvm.gc.collections.count:
+    description: The number of garbage collections performed by the JVM.
+    unit: "{collections}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [generation]
+    enabled: true
+  elasticsearch.node.jvm.gc.collections.elapsed:
```

```suggestion
  elasticsearch.node.jvm.gc.collections.time:
```

I think this is a [time metric](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#instrument-naming), so should end with `.time`. Since "elapsed" is an implicit feature of time, we can probably just replace the last part.

---

I think it's a pretty clear time metric, so the other integration has it wrong and will have to change. Not worth doubling down on the wrong one in my opinion.